add landing page task solution

siefimov opened the pull request:
[DEMO LINK](https://siefimov.github.io/layout_landing-page/)

etojeDenys reviewed (changes requested):
1. disable textarea scaling
<img width="986" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/70220667/c2c13243-223a-4048-8a82-15394cc36951">

2. disable scroll when menu is open

DarkMistyRoom reviewed (changes requested):
Looks good, but let's make it better:
- remove the horizontal scroll 
<img width="1680" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/98764641/3e9a4cf7-c8d2-4480-8bb0-54f2b88642bf">

- the header looks higher on the mockup
<img width="911" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/98764641/a64e7d82-b464-43f6-a974-d55aab6a848a">

- I can't open the menu on the mobile screen. When I click on the icon the tel link opens

- add scroll to the menu to make it work fine in landscape mode
<img width="686" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/98764641/080eb364-de7a-43f9-9b1f-d055d857e405">

siefimov commented:
I've made the appropriate changes:
- removed the horizontal scroll;
- made header higher on the desktop mode;
- fixed click on th menu icon to open menu;
- added scroll to the menu to ensure it works fine in landscape mode;

vadiimvooo reviewed (changes requested):
Good job. Just a few changes:
1. The header should be 100vh instead of fixed height
![image](https://github.com/mate-academy/layout_landing-page/assets/108701488/e6f6423a-ff6f-4f82-8c33-3f154f78fda3)
2. Width of button should be the same as textearea
![image](https://github.com/mate-academy/layout_landing-page/assets/108701488/840894b5-8bef-43b7-b5d8-27e4d65f7d97)

siefimov commented:
I've added changes to make header height 100vh, but I wonder how it alings with requirements for image to have aspect ration 1260 x 720:

![image](https://github.com/mate-academy/layout_landing-page/assets/17623764/a14329ff-78f5-47b3-98a2-bd6eef66067a)

IvanFesenko reviewed (changes requested):
Good job, almost done.
- header should have 100vh height
- Remove autocomplete styles
![2024-05-09_22h27_58](https://github.com/mate-academy/layout_landing-page/assets/61904995/e4031481-96c4-4409-8eae-2005b15f0415)

VitaliyBondarenko1982 reviewed (approved):
Looks good.
